src/styles/questionnaire.scss

```diff
@@ -200,3 +201,76 @@
     }
   }
 }
+
+.location-question-container {
+  .answer-form-container {
+    display: flex;
+
+    .rrs__multiselect-label__badge {
```

Comment:
what is this rrs stuff?

Reply:
It's the name of the class generated by the module we are using to fix the select menu